packages/headless/src/user.ts

```diff
@@ -49,6 +49,10 @@ export function useRegisterUser({
   });
 }
 
+/**
+ * This hook updates both the topics that the user is interested in and his profile
+ * Data
```

Comment:
We can improve the description to look something like this. 

- No need to mention that this is a hook :) 

```suggestion
 * Update all user data in one mutation. 
```